charts/rekor/templates/backfillredis/backfill-job.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.backfillredis.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "rekor.backfillredis.fullname" . }}
```

Comment:
`rekor.backfillredis.fullname` needs to be defined in `_helper.tpl`

Reply:
oops, I didn't save it.